Modules/Office365DSC/DSCResources/MSFT_SCSupervisoryReviewPolicy/MSFT_SCSupervisoryReviewPolicy.schema.mof

```diff
@@ -0,0 +1,9 @@
+[ClassVersion("1.0.0.0"), FriendlyName("SCSupervisoryReviewPolicy")]
+class MSFT_SCSupervisoryReviewPolicy : OMI_BaseResource
+{
+    [Key, Description("The Name parameter specifies the unique name for the supervisory review policy. The name can't exceed 64 characters. If the value contains spaces, enclose the value in quotation marks.")] String Name;
```

Comment:
If this parameter has a length restriction, we should be using the ValidateLength parameter validation. Or at least check the length of the input in the code.

And what do you mean about the comment when using spaces?

Reply:
The comment is taken from the official Docs.Microsoft.com article. Added ValidateLength checks